### PR TITLE
[FEAT] Notification pop up is completed

### PR DESCRIPTION
**Pull request by RohanDarade**
Currently the position of popup is set fixed and height is auto

### CODE REVIEW

File: src/components/Notification/Notification.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                <div className="min-h-[8px] min-w-[8px] max-h-[8px] max-w-[8px] rounded-full bg-blue-500 mr-4"></div>
+                <a href={item.link}>
+                  <div className="text-sm ">{item.message}</div>
+                </a>
+              </div>
+            ))}
+          </div>
+        </div>
+      ) : (
+        <div className="fixed top-56 z-10 right-0 mx-8 rounded-lg">
```
Comment (JayeshVP24) [inline src/components/Notification/Notification.jsx:65]: give white background and radius full , that bell  is hard to find. also add hover effects

File: src/components/Notification/Notification.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M6 18L18 6M6 6l12 12"
+                />
+              </svg>
+            </button>
+          </div>
+          <div>
+            {notification.map((item) => (
+              <div className="border-t-[1px] px-6 flex flex-row py-3 min-h-fit justify-start border-[#E4E7EC] items-baseline">
```
Comment (JayeshVP24) [inline src/components/Notification/Notification.jsx:55]: add hover effects here as well

border-t is equivalent to border-t-[1px[

File: src/layouts/Layout.astro [hunk 3]
```diff
@@ -55,7 +55,7 @@ import "../styles.css";
     <meta name="msapplication-TileColor" content="#00aba9" />
     <meta name="theme-color" content="#ffffff" />
 	</head>
-	<body>
+	<body class="overflow-x-hidden max-w-screen mx-auto" >
```
Comment (JayeshVP24) [inline src/layouts/Layout.astro:58]: why did u do this? can u explain?

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/layouts/Layout.astro (hunk 3 above)
Comment (RohanDarade) [reply src/layouts/Layout.astro]: I've no clue when did this happended, as far as ik layout and certified by section me I didn't made any changes

Comment (RohanDarade) [reply src/layouts/Layout.astro]: image
This was the ss of the git status i did just after git add

File: src/components/Notification/Notification.jsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+              </div>
+            ))}
+          </div>
+        </div>
+      ) : (
+        <div className="fixed top-56 z-10 right-0 mx-8 rounded-lg">
+          <div className="px-3">
+            <button
+              onClick={onClickOpen}
+              className="bg-white p-3 rounded-full hover:drop-shadow-xl"
```
Comment (JayeshVP24) [inline src/components/Notification/Notification.jsx:72]: give shadow-lg or xl maybe and change bg color on hover

**Review by JayeshVP24 — DISMISSED** [review]

**Review by kamalika0363 — APPROVED** [review]

**Pull request merged**